.github/workflows/test-pr.yml

```diff
@@ -1,28 +1,11 @@
 name: Test Azimuth deployment
 
 on:
-  pull_request:
+  # Trigger on completion of pull request Lint workflow 
+  workflow_run: 
+    workflows: [Lint]
```

Comment:
I think this approach will cause this workflow to trigger every time the Lint workflow is run, without running in the context of the branch/PR that the triggering Lint workflow is run, e.g. Lint workflow runs in branch `feat/xyz` and this workflow (once pushed to `main`) will be triggered and run in the `main` context, not in the `feat/xyz` context.

Reply:
I think this approach needs a rethink - the lint steps need to be part of the same workflow as the Azimuth test steps, or we can't make one a dependency of the other.